include/aikido/planner/kunzretimer/KunzRetimer.hpp

```diff
@@ -44,6 +44,14 @@ std::unique_ptr<aikido::trajectory::Spline> computeKunzTiming(
 class KunzRetimer : public aikido::planner::TrajectoryPostProcessor
 {
 public:
+  /// Kunz postprocessor parameters used with the main `PostProcessorParams`
+  /// struct.
+  struct Params
+  {
+    double mMaxDeviation = 1e-2;
```

Comment:
(1) I think this should have a reference to (or repeat of) the documentation from the constructor (also, the constructor should contain *units* if they are relevant).
(2) We should make this a `constexpr` like the Hauser parameters. Additionally, if we have any rationale for why a given default is used, that can be added there.

Reply:
Done!